src/Display/Tabs/Tabs.tsx

```diff
@@ -49,7 +49,8 @@ const Tabs: Tabs = (
                     `tab-${tabLabel}`,
                     { active: activeTabOrIndex === tabLabel },
                     `${alignment}-tab`,
-                    `${variant}`
+                    `${variant}`,
+                    'tab'
```

Comment:
Was the className `'tab'` present before? If not we can leave it out.

Reply:
No it wasn't.
Removed it! :smile: